castle/migrations/0029_challenge.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I am happy with the migration code being in source control, if only because that is how I operated when I was doing this by myself.

Having said that, I can also see the argument that there is no need to check in files that are generated automatically by Django in response to changes in models.py.

Maybe we should have this discussion elsewhere (maybe raise a separate issue), and come to a decision one way or another.